### PR TITLE
No avatar uploads

### PR DESCRIPTION
**Pull request by nwinter**
Disable all file uploads for regular users, tell users how to set up Gravatars instead on account settings page, don't show portrait or name edit on OptionsView (the in-game options modal), and fall back to the user's hero portrait if no Gravatar. This should prepare us to delete all user-uploaded profile pictures whenever convenient to save space.

### CODE REVIEW

**Comment by nwinter** [comment]
I accidentally put another unrelated bug fix on this branch, too.

File: scripts/analytics/projectShares.js [hunk 1]
```diff
@@ -116,6 +116,13 @@ co(function*() {
   }
   console.log("RESULT: ", (totalViews / _.size(levelSessionViews)).toFixed(3), 'views per completed shareable project.')
 
+  var bestProjects = [];
+  for(var sessionId in levelSessionViews) {
+    bestProjects.push({views: levelSessionViews[sessionId].views, sessionId: sessionId});
+  }
+  bestProjects = _.sortBy(bestProjects, function(project) { return -project.views; });
+  console.log(bestProjects.slice(0, 50));
+
```
Comment (sderickson) [inline scripts/analytics/projectShares.js]: Unrelated changes?

File: server/handlers/user_handler.coffee [hunk 2]
```diff
@@ -497,8 +497,6 @@ UserHandler = class UserHandler extends Handler
       photoURL = document?.get('photoURL')
       if photoURL
         photoURL = "/file/#{photoURL}"
```
Comment (sderickson) [inline server/handlers/user_handler.coffee]: Shouldn't we stop using photoURL on client and server, and mark the property deprecated?

Comment (nwinter) [inline server/handlers/user_handler.coffee]: I was going to leave it in in case we start letting users choose avatars from a list, then this could would be useful again. This will also let people continue to use their existing avatars until we actually go through and delete them after we have a backup.

File: scripts/analytics/projectShares.js (hunk 1 above)
Comment (nwinter) [inline scripts/analytics/projectShares.js]: My git hygiene is so great... I didn't mean to commit that, but since I did, I just finished fixing it up so that it works again and also outputs best projects.

File: server/handlers/user_handler.coffee (hunk 2 above)
Comment (sderickson) [inline server/handlers/user_handler.coffee]: But then people with existing avatars can't change their image or use gravatar.

Comment (nwinter) [inline server/handlers/user_handler.coffee]: I expect a very small percentage of our customers will bother setting up a Gravatar, and our existing active user avatar base is large compared to the number of people who already have one and will need to change it in the next X time period until we get rid of them.
